templates/components/itilobject/fields/status.html.twig

```diff
@@ -36,7 +36,8 @@
 {% if canupdate %}
    {% set status_field %}
       {{ item.dropdownStatus({
-         'value': item.fields['status'],
+         'value': item.input['status']|default(item.fields['status']),
+         'value_calculation': item.fields['status'],
```

Comment:
Unless I am wrong, `item.input` would be set only when `prepareInputFor...` returns false for a form submitted from UI, and, in this case, value was a valid value in the invalidated form. Not sure we have to introduce this `value_calculation`.

Reply:
ping @cconard96

Reply:
The input array is filled when saved values are being restored after some form input refreshes the page.
That means the displayed value will try to be the new selection if it exists (input) or the item's current one (fields).
The value used to calculate which statuses are allowed, will always use the item's current one (fields).